syncpack.config.cjs

```diff
@@ -186,7 +186,6 @@ module.exports = {
 				"@fluidframework/common-utils",
 				"@fluidframework/eslint-config-fluid",
 				"@fluidframework/protocol-definitions",
-				"@fluidframework/test-tools",
```

Comment:
Now part of the client release group, so doesn't need to be specified here (syncpack complains if it is, not sure exactly why; says all consumers are using `workspace:~` but the package itself says `2.11.0` (the actual version in package.json)).